index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <div class="header">
+    <h1>Hello Microverse</h1>
+  </div>
```

Comment:
In order to give semantic meaning to our tags, it is recommended to use [semantic HTML tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics).

- [ ] In this case, it is better to replace `<div></div>` with `<header></header>`.